### PR TITLE
docs(misc): document model export and claim/unclaim with body

### PR DESCRIPTION
**Pull request by daddykotex**
updated the doc and added the event type origin

### CODE REVIEW

**Comment by daddykotex** [comment]
I updated line 43 and 43, to document the optional body

I also documented the model export

File: README.md [hunk 1]
```diff
@@ -35,31 +35,32 @@ exception will be thrown so you know what happened.
...
+| ------- | ---------------------------------------------------- | ------------------------------------------------------------------------| ----------------------------------------          |
+| Owners  | `create(owner)                                     ` | create a new owner                                                      | [simple_workflow.py](examples/simple_workflow.py) |
+|         | `update(username, owner)                           ` | update an existing owner                                                |                                                   |
+|         | `claim(username, device_id, optional_body)         ` | link an existing object to an existing owner                            | [simple_workflow.py](examples/simple_workflow.py) |
+|         | `unclaim(username, device_id, optional_body)       ` | unlink an existing object from an existing owner                        | [simple_workflow.py](examples/simple_workflow.py) |
+|         | `batch_claim(claims)                               ` | claim a batch of object                                                 | [simple_workflow.py](examples/simple_workflow.py) |
```
Comment (jtheoof) [inline README.md]: objects (plural)

File: README.md [hunk 2]
```diff
@@ -35,31 +35,32 @@ exception will be thrown so you know what happened.
...
+|         | `batch_unclaim(unclaims)                           ` | unclaim a batch of object                                               | [simple_workflow.py](examples/simple_workflow.py) |
```
Comment (jtheoof) [inline README.md]: objects (plural)

Comment (daddykotex) [inline README.md]: Let me recheck it, ty

Comment (daddykotex) [inline README.md]: done

File: README.md (hunk 1 above)
Comment (daddykotex) [inline README.md]: done

**Comment by daddykotex** [comment]
I fixed a few more typos and updated titles

**Comment by jtheoof** [comment]
👍

**Comment by jtheoof** [comment]
Just waiting for @mvonthron review if he dares :)

**Comment by mvonthron** [comment]
LGTM

**Pull request merged**